modules/nf-core/ascat/tests/main.nf.test

```diff
@@ -0,0 +1,42 @@
+nextflow_process {
+
+    name "Test Process ASCAT"
+    script "../main.nf"
+    process "ASCAT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ascat"
+
+    test("ascat") {
+
+        config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id: 'test', single_end:false ],
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam_bai'], checkIfExists: true)
+                ]
+                input[1] = file(params.test_data['homo_sapiens']['illumina']['test2_allele_specific_recal'], checkIfExists: true)
+                input[2] = []
```

Comment:
Did you check if `path(loci_files)` is optional aswell? I know we discussed that on slack but maybe that is why the nf-test does not run?

Reply:
It shouldn't be optional but for some reason pytest works without loci and allele files. I'm trying to figure out why.

Reply:
I also would like to run nf-test using `loci_files` and `allele_files` but they are not in **test_data**

Reply:
> It shouldn't be optional but for some reason pytest works without loci and allele files. I'm trying to figure out why.

Got it now, pytest only executed stub test: `nextflow run ./tests/modules/nf-core/ascat -entry test_ascat -c ./tests/config/nextflow.config -stub-run`

Reply:
Ok then we'll need to search for alternatives for those files [here](https://github.com/nf-core/modules/blob/master/tests/config/test_data.config) here or if there are none add them.

Reply:
Or wait, loci and allele are only needed for WGS right? So maybe its enough if we specify the test for WXS and then add a bed file?